package/Dockerfile.dapper-base

```diff
@@ -28,7 +28,7 @@ ENV GOLANG_ARCH_amd64=amd64 GOLANG_ARCH_arm=armv6l GOLANG_ARCH=GOLANG_ARCH_${ARC
 # goimports      | code formatting
 # make           | OLM installation
 # findutils      | validate (find go packages)
-# shflags        | shell script flags
+# shflags        | shell script flags (not in Fedora 32)
```

Comment:
```suggestion
```

I don't see a reason to keep this mention here (it'll live on in git history anyway)